src/main/java/nextstep/courses/domain/Student.java

```diff
@@ -23,6 +24,14 @@ public Long getSessionId() {
         return sessionId;
     }
 
+    public boolean isApproved(){
+        return ApprovalState.isApproved(approvalState);
```

Comment:
```suggestion
        return approvalState.isApproved();
```
위와 같이 구현하는 습관을 들이는 것은 어떨까?